src/parcsr_ls/par_amg_setup.c

```diff
@@ -1423,7 +1419,6 @@ hypre_BoomerAMGSetup( void               *amg_vdata,
                if (SN) hypre_ParCSRMatrixDestroy(SN);
                if (AN) hypre_ParCSRMatrixDestroy(AN);
                //hypre_TFree(CF_marker, HYPRE_MEMORY_HOST);
-               hypre_TFree(coarse_pnts_global, HYPRE_MEMORY_HOST);
```

Comment:
Why not freeing it?

Reply:
For the same reason as in https://github.com/hypre-space/hypre/pull/392#discussion_r657303905